rust/src/connection/network/transmitter/rpc.rs

```diff
@@ -156,19 +141,13 @@ impl RPCTransmitter {
             }
 
             Request::UsersAll => rpc.users_all(request.try_into_proto()?).await.map(Response::from_proto),
-            Request::UsersContain { .. } => {
-                rpc.users_contain(request.try_into_proto()?).await.map(Response::from_proto)
+            Request::UsersContains { .. } => {
+                rpc.users_contains(request.try_into_proto()?).await.map(Response::from_proto)
             }
             Request::UsersCreate { .. } => rpc.users_create(request.try_into_proto()?).await.map(Response::from_proto),
             Request::UsersDelete { .. } => rpc.users_delete(request.try_into_proto()?).await.map(Response::from_proto),
             Request::UsersGet { .. } => rpc.users_get(request.try_into_proto()?).await.map(Response::from_proto),
-            Request::UsersPasswordSet { .. } => {
-                rpc.users_password_set(request.try_into_proto()?).await.map(Response::from_proto)
-            }
-
-            Request::UserPasswordUpdate { .. } => {
-                rpc.user_password_update(request.try_into_proto()?).await.map(Response::from_proto)
-            }
+            _ => todo!()
```

Comment:
I'd put explicit todos  with names of requests which are not implemented, especially if we are already releasing like:
`Request::XXX => todo!("XXX")`. Using `_` will prevent the static checks to highlight new requests forgotten to be propagated everywhere. Especially if there is just a small number of these requests. 
Not really an issue if you're sure that you're going to cover it in the next PR.

Reply:
Turns out it's no longer needed since all the cases have been implemented! Removed in 415ecb92e629877603e64f85315cbebc28d7b96c

Reply:
That's another issue of using `_ => ...`! =)